reposcan/exporter.py

```diff
@@ -336,25 +336,16 @@ def dump_cves(self, dump):
     def dump_modules(self, dump):
         """Select module information"""
         with self._named_cursor() as cursor:
-            if self.errata_ids:
```

Comment:
this if is still needed, select will fail in case of no errata in repo

Reply:
Will change this